.travis.yml

```diff
@@ -1,6 +1,15 @@
 sudo: false
-language: haskell
+addons:
+  apt:
+    sources:
+        - hvr-ghc
+    packages:
+        - libgmp-dev
+        - ghc-8.0.1
```

Comment:
Can you explain why the new ghc is required? Does that mean aws will no longer build on older compilers?